modules/osdk-ansible-k8s-module-testing-locally.adoc

```diff
@@ -13,12 +13,12 @@ local machine as opposed to running and rebuilding the Operator each time.
 . Initialize a new Ansible-based Operator project:
 +
 ----
-$ operator-sdk new --type ansible --kind Foo --api-version foo.example.com/v1alpha1 foo-operator
+$ operator-sdk new --type ansible --kind foo --api-version foo.example.com/v1alpha1 foo-operator
```

Comment:
@sheriff-rh We don't need to modify the `Foo` here. The `foo` doesn't work, see: 
```console
Error: value of --kind must start with an uppercase letter
```